app/components/post_list/index.tsx

```diff
@@ -352,7 +352,6 @@ const PostList = ({
                     keyboardShouldPersistTaps='handled'
                     keyExtractor={keyExtractor}
                     initialNumToRender={INITIAL_BATCH_TO_RENDER + 5}
-                    listKey={`postList-${channelId}`}
```

Comment:
for my understanding, why was this one removed ?

Reply:
I did not find a use case for this to be kept